app/src/main/java/org/sil/gatherwords/EntryActivity.java

```diff
@@ -625,6 +625,8 @@ protected Void doInBackground(Void... v) {
             // Insert new Word
             Word newWord = new Word();
             newWord.sessionID = sessionID;
+            newWord.updatedAt = new Date();
+            Log.d("EntryActivity", "I added a Date to a new Word!");
```

Comment:
For the tag parameter, you can pass `TAG` (convenience variable most classes now have).

Printing to the debug log is fine.  However, I think in this case the debug message is not significant enough that other devs would make use of it.  I would recommend removing this and your other debug message.